images/aws_upload.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash 
+
+set -e
+
+disk="$1"
+s3_bucket="cos-images"
+disk_name="cOS-Vanilla"
+disk_desc="cOS Vanilla Image"
+
+[ -f "${disk}" ] || exit 1
+
+echo "Uploading to S3 Bucket: ${s3_bucket}"
+aws s3 cp "${disk}" "s3://${s3_bucket}" --metadata "Name=${disk_name}"
+
+echo "Import snapshot to EC2"
+task_id=$(aws ec2 import-snapshot --description "${disk_desc}" \
+    --disk-container "Description=${disk_desc},Format=raw,UserBucket={S3Bucket=${s3_bucket},S3Key=$(basename ${disk})}" \
+    --tag-specifications "ResourceType=import-snapshot-task,Tags=[{Key=Name,Value=${disk_name}}]" \
```

Comment:
can you also tag it with `Flavor: recovery` ? This is because in the future if we want to introduce auto deletion, we should probably have different policies per flavor image (we may just want to have only 1 recovery at the time) so we can filter by this flavor tag easily :D

Reply:
Also version would be nice to have :+1:

Reply:
These tags are only for the `import-snapshot` task... I thought they would apply for the resulting snapshot but they don't, I just did not remove them, it doesn't hurt and provides some information in case there is anyone watching

Reply:
ahh indeed!